### ISSUE TITLE
Visual Studio Code powershell defect

### ISSUE THREAD

**Issue by YorkEarwaker** [PowerShell/PSReadLine#4243]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
Oops, something went wrong.  Please report this bug with the details below.
Report on GitHub: https://github.com/lzybkr/PSReadLine/issues/new
-----------------------------------------------------------------------
Last 100 Keys:
 Space & Space ' C : \ P r o g r a m Space F i l e s \ E c l i p s e Space A d o p t i u m \ j d k - 2 1 . 0 . 4 . 7 - h o t s p o t \ b i n \ j a v a . e x 
e ' Space ' - a g e n t l i b : j d w p = t r a n s p o r t = d

Exception:
System.ArgumentOutOfRangeException: The value must be greater than or equal to zero and less than the console's buffer size in that dimension.
Parameter name: left
Actual value was -2.
   at System.Console.SetCursorPosition(Int32 left, Int32 top)
   at Microsoft.PowerShell.PSConsoleReadLine.ReallyRender(RenderData renderData, String defaultColor)
   at Microsoft.PowerShell.PSConsoleReadLine.ForceRender()
   at Microsoft.PowerShell.PSConsoleReadLine.Insert(Char c)
   at Microsoft.PowerShell.PSConsoleReadLine.SelfInsert(Nullable`1 key, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.ProcessOneKey(ConsoleKeyInfo key, Dictionary`2 dispatchTable, Boolean ignoreIfNoAction, Object arg)
   at Microsoft.PowerShell.PSConsoleReadLine.InputLoop()
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics)
-----------------------------------------------------------------------
```

### Screenshot

![image](https://github.com/user-attachments/assets/aed1bdc1-2cd9-49b7-a7ce-a4423de5d6c2)

### Environment data

```console
PS Version: 5.1.19041.5007
PS HostName: ConsoleHost
PSReadLine Version: 2.0.0-beta2
PSReadLine EditMode: Windows
OS: 10.0.19041.1 (WinBuild.160101.0800)
BufferWidth: 157
BufferHeight: 38
```

### Steps to reproduce

Jena and Fuseki install localhost. 
Apache Jena 5.2.0.
Coding in Visual Studio Code.
Please see in first instance:
https://github.com/YorkEarwaker/Data/blob/main/metadata/rdf-code-samples/jena/src/fuseki_hw/tomcat_fuseki_env.bat
https://github.com/YorkEarwaker/Data/blob/main/metadata/rdf-code-samples/jena/jena_env.bat
https://github.com/YorkEarwaker/Data/tree/main/metadata/rdf-code-samples/jena

### Expected behavior

?
The bug is reported as requested.

Expect the latest powershell to have been installed with latest OS update. Which is the powershell Visual Studio Code uses.
Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine) *

Have not done this as not sure where to look for pinned issues.
Search the existing issues, especially the pinned issues. *

### Actual behavior

?

**Comment by YorkEarwaker** [comment]
Dear PowerShell/PSReadLine,
I updated PowerShell to 2.3.6 . The latest version. Windows system32.
Rebooted machine. Reopened Visual Studio Code. Shell integration failed to activate.
What next?


Yours,York_________________________________________________York Earwaker | London UK | Mb: +44 (0)77 8990 7102_________________________________________________ 

    On Wednesday 30 October 2024 at 16:55:11 GMT, github-actions[bot] ***@***.***> wrote:  
 
 


This issue was already fixed (see #1306). Please upgrade to the 2.3.5 version of PSReadLine from PowerShell Gallery.
See the upgrading section for instructions. Please let us know if you run into the same issue with the latest version.

**Comment by daxian-dbw** [comment]
@YorkEarwaker Shell integration is not covered in this repository. You can ask about it in https://github.com/PowerShell/vscode-powershell.
